policyengine_us/parameters/gov/irs/income/exemption/overtime/computer_salary_threshold.yaml

```diff
@@ -0,0 +1,10 @@
+description: The US classifies as exempts from overtime salaried individuals working in computer science related positions if they have an hourly salary level of or above this value.
+metadata:
+  unit: currency-USD
+  period: year
+  label: Computer science salary threshold
```

Comment:
Labels lowercase please- droppable mid-sentence (so that the app doesn't have bad grammar when describing 'set the Computer science salary threshold to X'.

Reply:
And other places